### DISCUSSION TITLE
DynamicStall file

### DISCUSSION THREAD

**Discussion by dkrgajjela** [xcompact3d/Incompact3d#133] [Q&A] [answered]
Can someone please explain what's the dynamic stall file in turbines example contain and where can i get that file?
Thank You.

**Comment by nbeb** [comment, marked as answer]
The dynamic stall file includes the coefficients to be used in the dynamic stall model (described in dynstall.f90). These vary with different airfoils. An example file is attached below.
[S826DS.txt](https://github.com/xcompact3d/Incompact3d/files/10069148/S826DS.txt)

**Reply by Sushmitharao46** [reply]
Could you please tell where can I get the files for different airfoils for different Reynolds numbers.
Thank You.

**Comment by dkrgajjela** [comment]
okay got it. Thank you @nbeb

**Comment by Sushmitharao46** [comment]
@nbeb  Could you please tell where can I get the files for different airfoils for different Reynolds numbers.
Thank You.
